### DISCUSSION TITLE
版本 1.8.x 预告

### DISCUSSION THREAD

**Discussion by lspriv** [lspriv/wx-calendar#130] [Ideas]
#### 1. 分离农历插件
农历插件不再作为内置插件。
#### 2. 控制月试图和日程视图下非本月日期的显隐。
当前在月份的前后两端是补全日期的，提供新的属性来控制这些日期是否显示。

**Comment by lspriv** [comment]
欢迎补充。

**Reply by hoythan** [reply]
希望能够支持 slot 让用户自定义

**Reply by hoythan** [reply]
希望支持关闭动画效果的功能

**Reply by lspriv** [reply]
> 希望支持关闭动画效果的功能

详细说说
